metadata-ingestion/src/datahub/ingestion/source/metadata/business_glossary.py

```diff
@@ -332,10 +333,10 @@ def get_mces_from_term(
             models.GlossaryTermKeyClass,
             models.BrowsePathsClass,
         ]
-    ] = []
+    ] = [valid_status]
```

Comment:
Would prefer this as `VALID_STATUS` to make it clear it's a global. Also, not familiar with this source at all, I assume we can't use the `auto_status_aspect` helper?

Reply:
you're right - switching to use the auto_status_aspect instead